pkg/ratelimit/http.go

```diff
@@ -13,18 +13,21 @@ var limitHeaders = []string{
 	"X-Ratelimit-Limit",
 	"Ratelimit-Limit",
 	"X-RateLimit-Requests-Limit", // Linear uses a non-standard header
```

Comment:
its weird to me we can't handle this via option parameters to uhttp or somehting.  centralizing this is... well, gonna be hard, and future error prone.

Reply:
We'll get 'em all eventually. Just one more http header and we'll have all of them covered. For sure this time. 😅

I agree that this setup is not ideal, but so far we only have two APIs that are wrong (Linear and Okta). If we have a 3rd one, then I think it's time to tweak this.